Dockerfile.template

```diff
@@ -1,3 +1,16 @@
+FROM multiarch/debian-debootstrap:armel-stretch-slim as build
+WORKDIR /tmp
+
+{% if pihole.build %}
```

Comment:
Why not move this up 2 more lines?  The build stage is also appearing in all of the other Dockerfiles which are not using it.

Reply:
I've fixed the issue where Armel is being used for all Dockerfiles. The reason that the build stage appears is to keep the build steps in Travis consistent. Without breaking the first stage build into a separate command, Travis times out because there is no output from `Dockerfile.py`.

If you have a better hacky or a preferred way, I'm all ears. :smile:

Reply:
Is the timeout because of no printing happening or because it is a single step taking so long?

We can add prints to Dockerfile.py if necessary and keep 100% of the build logic in there.  In python it'd be easy to add `if armel : --target build` & skip for the rest.